src/app/tabs/theme-module.js

```diff
@@ -26,7 +26,7 @@ const themes = [
 const profile = {
   name: 'theme',
   events: ['themeChanged'],
-  methods: ['switchTheme', 'getThemes', 'currentTheme'],
+  methods: ['switchTheme', 'getThemes', 'currentTheme', 'fixInvert'],
```

Comment:
`fixInvert`  looks like a method internal to Remix and should not be called by other plugins.
btw `switchTheme` neither I think

Reply:
I made it external in purpose. why not? Sokrates, for example, could use it for their Logo.

Reply:
yeah but that is more an helper that could be made available through the remix plugin than an API function.

Reply:
what do you mean by "remix plugin" ?

Reply:
I meant the remix-plugin library

Reply:
"theme" is not a part of the standard plugin api. It only apply for remix-ide. I don't have any hard feeling about having `switchTheme` as it doesn't look dangerous to me (maybe devs would like to build custom theme for remix in the future).
Concerning the `fixInvert` it wouldn't work for external plugins anyway, as it expects an Img HTMLElement, and large objects like that cannot be exchanges between Iframes.

Also I think it should be more a global css class than a javascript utils : 
```css
.dark-theme img.invert {
  filter: invert(1);
}

.light-theme img.invert {
  filter: invert(0);
}
```

Reply:
some images are in color and some shouldn't change the invert.

Reply:
and if I pass id it will work. but does it make sense at all? do you think it worth to have such a function? Is it obvious that `invert` can be used?

Reply:
@GrandSchtroumpf

Reply:
I think it doesn't make so much sense to have an "invert" function. It would make more sense to provide a list of classes we are using in the application for developers to create their own theme.

Reply:
ok then i'll remove it